cogs/welcome/welcome.py

```diff
@@ -537,6 +608,37 @@ def check(message: discord.Message):
         await self.config.guild(ctx.guild).get_attr(key).set(greetings)
         return
 
+    # [p]welcomeset greetings addimg
+    @greetings.command(name="addimg")
+    async def imgAdd(self, ctx: Context):
+        """adds the attached image to the pool of random based images used to generate custom welcome images. Attaches only the first image attached.
```

Comment:
nit: Capitalize first letter and make this singular verb to follow other commands.